### PR TITLE
Portfolio: Setup and mobile first

### PR DESCRIPTION
**Pull request by jcmunav63**
**Description**
For the first milestone in the process of creating your portfolio website, you will:

- Set up a new repository and prepare it for development using best practices (e.g. linters).
- Build the first 2 sections of the mobile website using the template you chose in the previous step.

**General requirements**
- Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config):
  - Linter should be set up correctly for this repository.
  - Linters result should be green.
  - There should be no custom changes to linters config.
- Make sure that you used the correct [GitHub Flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md):
  - The README file should be descriptive.
  - Commit messages should be meaningful.
  - Pull Request should have a descriptive title explaining what changes are inside.
  - Pull Request should have a short summary describing introduced changes in more detail.
- Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md):
  - The branch name should describe a feature or action (e.g. 'created_sidebar' or 'added_comments_section').
  - The Pull Request is made from a 'feature branch' into the main branch.

**HTML/CSS requirements**
Follow our list of [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

**Project requirements**
- For this project, you should use the [HTML&CSS linter](https://github.com/microverseinc/linters-config/tree/master/html-css)
- The 2 sections you need to build for this project are:
   * The toolbar (or header).
   * The headline section (right after the header).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./css/styles.css">
+  <title>Portfolio - Setup and Mobile First</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;400;700&family=Roboto&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header class="toolbar">
+    <div class="container-2">
```
Comment (tufoinnkuo10) [inline index.html:15]: - [ ] Great job so far. Please kindly fix the fixed width you have given your portfolio to make it responsive right up to the desktop version(768px). Nice work.
Screenshot (1101)_LI

File: index.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        <div class="logo"><a href="#home"><p class="logo-txt poppins">My Logo</p></a></div>
+        <nav class="menu"><img class="menu-im" alt="Menu" width="24px" height="24px" src="images/menu.svg"></nav>
+    </div>
+  </header>
+  <section class="headline poppins">
+    <h1 class="name">I'm Juan Carlos. Glad to see you!</h1>
+    <p class="desc">I'm a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+    <p class="connect">LET'S CONNECT</p>
```
Comment (tufoinnkuo10) [inline index.html:23]: - [ ] please kindly make this an `hx(2/3)` tag here as it is a subtitle. This way will be semantically correct and the screen readers and other technologies will thank you

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @jcmunav63 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ figma design followed

Required Changes ♻️
Check the comments under the review

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by bienvenuushindi — APPROVED** [review]
# Approved 👏  

## Hi @jcmunav63      👋, 

Well done implementing the requested changes 👏

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

**Pull request merged**